po/es.po

```diff
@@ -1639,29 +1633,29 @@ msgid "`%s' is not a valid key command"
 msgstr "`%s' no es un atajo de teclado válido"
 
 #: src/keymap.cpp:775
-#, fuzzy, c-format
+#, c-format
 msgid "failed to parse operation sequence for %s"
-msgstr "Error al cargar la fuente «%s»: %s"
+msgstr "no se pudo analizar la secuencia de operaciones para %s"
 
 #: src/keymap.cpp:793
-#, fuzzy, c-format
+#, c-format
 msgid "`%s' is not a valid operation"
-msgstr "`%s' no es un contexto válido"
+msgstr "`%s' no es una operación válida"
 
 #: src/matcherexception.cpp:15
 #, c-format
 msgid "attribute `%s' is not available."
-msgstr "el atributo `%s' no está disponible."
+msgstr "el atributo '%s' no está disponible."
```

Comment:
Looks like an erroneous edit. Is there a rationale?

(We intend to bring some order to our use of quotes, #901, but it hasn't happened yet.)

Reply:
No, it is indeed an erroneous edit. Probably I got confused by other strings where the format was '%s'.

Now that I re-check, there's also a wrong quote in line 835 https://github.com/newsboat/newsboat/pull/1961/commits/4ea55229f6326f98e15258effdcfdccd494d94a2#diff-e489678c415f61892616a7f3ce50c4ffc843c0e1de965c59152a71fbdb388a49R835

 ` and ' are pretty similar visually, there is little difference, so I didn't spot those errors. My apologies.